capstone/src/main/java/com/google/univiz/api/SearchResourceImpl.java

```diff
@@ -0,0 +1,24 @@
+package com.google.univiz.api;
+
+// TODO: imports
+//
```

Comment:
nit - this TODO format isn't correct, and doesn't seem to be needed...

Reply:
yes indeed! removed!